modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCTextRunImpl.java

```diff
@@ -23,10 +23,11 @@
  * questions.
  */
 
-package com.sun.webkit.graphics;
+package com.sun.webkit.prism;
```

Comment:
The package still doesn't match the location: com.sun.webkit.prism != com.sun.javafx.webkit.prism

Reply:
Sorry for the silly typo again. Fixed it in the latest commit. 

Thanks Kevin.